exec/csnex/Support.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace csnex
+{
+    public class Support
+    {
+        static public List<string> Paths;
+
+        public Support(string source_path)
+        {
+            Paths = new List<string>();
+            // Always check current directory;
+            Paths.Add(string.Format(".{0}", Path.DirectorySeparatorChar));
+            // Also check the Neon executable path.
+            Paths.Add(string.Format("{0}{1}", source_path, Path.DirectorySeparatorChar));
+            // Then check the cwd.
+            Paths.Add(Environment.CurrentDirectory);
+
+            // Get paths from %NEONPATH% environment variable, if there is one.
+            string ne = Environment.GetEnvironmentVariable("NEONPATH");
+            if (ne != null) {
+                string[] evpaths = ne.Split(Path.PathSeparator);
+                foreach (string s in evpaths) {
+                    // Don't bother pushing empty paths.
+                    if (s != null && s.Length > 0) {
```

Comment:
Under what conditions would `Split()` return a `null` in the array? I don't see that mentioned anywhere in the documentation. You do not need to handle the `null` case if it is not documented to happen.

Reply:
Removed != null check.

Reply:
The check still seems to be there?